app/environments/brassbirmingham/brassbirmingham/envs/classes/buildings/enums.py

```diff
@@ -0,0 +1,16 @@
+from enum import Enum
+
+
+class BuildingName(Enum):
+    goods = "goods"
+    cotton = ("cotton",)
+    pottery = "pottery"
+    oil = "oil"
+    coal = "coal"
+    beer = "beer"
+    iron = "iron"
+
+
+class BuildingType(Enum):
```

Comment:
Arguably "BuildingType" could mean the same thing as "BuildingName". Idk if there's a better word for these two